### PR TITLE
[Bots][Addedinlineimagesection][1466405]

### PR DESCRIPTION
**Pull request by v-rodriguezd**
Added Fetching inline images from message section with code and changed basic example C# code and added image and hyperlink. Please review.

### CODE REVIEW

File: msteams-platform/bots/how-to/bots-filesv4.md [hunk 1]
```diff
@@ -121,7 +121,7 @@ The following table describes the content properties of the attachment:
 
 #### Invoke activity when the user accepts the file
 
-An invoke activity is sent to the bot if and when the user accepts the file. It contains the OneDrive for Business placeholder URL that the bot can then issue a `PUT` into to transfer the file contents. For information on uploading to the OneDrive URL, see [Upload bytes to the upload session](/onedrive/developer/rest-api/api/driveitem_createuploadsession#upload-bytes-to-the-upload-session).
+An invoke activity is sent to the bot if and when the user accepts the file. It contains the OneDrive for Business placeholder URL that the bot can then issue a `PUT` into to transfer the file contents. For information on uploading to the OneDrive URL, see [upload bytes to the upload session](/onedrive/developer/rest-api/api/driveitem_createuploadsession#upload-bytes-to-the-upload-session).
```
Comment (KirtiPereira) [inline msteams-platform/bots/how-to/bots-filesv4.md:124]:
```suggestion
An invoke activity is sent to the bot if and when the user accepts the file. It contains the OneDrive for Business placeholder URL that the bot can then issue a `PUT` to transfer the file contents. For information on uploading to the OneDrive URL, see [upload bytes to the upload session](/onedrive/developer/rest-api/api/driveitem_createuploadsession#upload-bytes-to-the-upload-session).
```

**Review by KirtiPereira — CHANGES_REQUESTED** [review]
@v-rodriguezd - please verify and commit suggestion.

**Comment by KirtiPereira** [comment]
@Wajeed-msft - kindly review / approve the PR. Thank you.

**Review by Wajeed-msft — APPROVED** [review]

**Review by KirtiPereira — APPROVED** [review]
Approve

**Review by surbhigupta12 — APPROVED** [review]
Approving the PR.

**Pull request merged**